service/Service/appsettings.json

```diff
@@ -269,6 +269,22 @@
         // Setting used only for country clouds
         "EndpointSuffix": "core.windows.net"
       },
+      "S3": {
```

Comment:
This needs to be named AWSS3, right? And the keys should be according to the latest rename by you @dluc 

I added this as part of configuration to enable running Kernel Memory as a service with S3 storage. Is there anything else that we need to change for that?

Reply:
yes AWSS3 would be ideal, thanks 👍